internal/cli/delete_test.go

```diff
@@ -0,0 +1,35 @@
+package cli
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/tilt-dev/tilt/pkg/apis/core/v1alpha1"
+)
+
+func TestDelete(t *testing.T) {
+	f := newServerFixture(t)
+	defer f.TearDown()
+
+	err := f.client.Create(f.ctx, &v1alpha1.Cmd{
+		ObjectMeta: metav1.ObjectMeta{Name: "my-sleep"},
+		Spec: v1alpha1.CmdSpec{
+			Args: []string{"sleep", "1"},
+		},
+	})
+	require.NoError(t, err)
+
+	out := bytes.NewBuffer(nil)
+	deleteCmd := newDeleteCmd()
+	deleteCmd.register()
+	deleteCmd.streams.Out = out
+
+	err = deleteCmd.run(f.ctx, []string{"cmd", "my-sleep"})
+	require.NoError(t, err)
+
+	assert.Contains(t, out.String(), `cmd.tilt.dev "my-sleep" deleted`)
```

Comment:
Might be worth an assertion that `f.client.Get()` returns not found after this

Reply:
done